.rubocop_todo.yml

```diff
@@ -321,6 +317,12 @@ Style/NumericPredicate:
     - 'lib/bundler/source/git/git_proxy.rb'
     - 'lib/bundler/source/path.rb'
 
+# Offense count: 192
+# Cop supports --auto-correct.
+# Configuration parameters: PreferredDelimiters.
+Style/PercentLiteralDelimiters:
```

Comment:
might want to enable this one and set them to what the project is currently using?